src/summary/components/UtilsViews.tsx

```diff
@@ -52,7 +52,7 @@ export const TaskView = ({
       Created By: {user.firstName} {user.lastName}
       {printAssignees && (
         <div>
-          {task.assignees.length < 0 && <h6>The task does have assignees</h6>}
+          {task.assignees.length < 1 && <h6>This task does not have assignees</h6>}
```

Comment:
Changing core code logic is out of scope of adding tests. Could you revert this change?